modules/cluster-logging-updating-logging.adoc

```diff
@@ -264,14 +264,8 @@ cronjob.batch/elasticsearch-rollover-infra
 +
 You should see the `elasticsearch-delete-\*` and `elasticsearch-rollover-*` indices.
 
-.Post update
+== Post-update tasks
```

Comment:
@mburke5678 Can you please cross-check this? There was a conflict here when cherry-picking and I want to be sure that this change properly aligns with your updates in https://github.com/openshift/openshift-docs/pull/24579/files#diff-20556f45a785d31a600aeb2f691b381c. Looks like that PR was not cherry-picked for 4.6, so please let me know if this change is incorrect. Thanks!

Reply:
Confirmed with Michael in Slack. :tada: